kolibri/core/assets/src/vue/core-base.vue

```diff
@@ -35,7 +35,7 @@
   const values = require('lodash.values');
   const responsiveWindow = require('kolibri.coreVue.mixins.responsiveWindow');
 
-  const PADDING = 32;
+  const PADDING = 16;
```

Comment:
Possible, I didn't see it used anywhere except the left padding though, so hopefully it's ok!